robots/pr-creator/updater/updater.go

```diff
@@ -46,7 +46,8 @@ func UpdatePR(org, repo, title, body, matchTitle string, gc updateClient) (*int,
 		return nil, fmt.Errorf("bot name: %v", err)
 	}
 
-	issues, err := gc.FindIssues("is:open is:pr archived:false in:title author:"+me+" "+matchTitle, "updated", false)
+	query := fmt.Sprintf("is:open is:pr archived:false author:%s in:title %s %s", me, matchTitle, extraQuery)
```

Comment:
I assume there's no harm in the extra "" when extraQuery is empty?

Reply:
Nope.
/hold cancel